src/components/ReactTable/ReactTable.tsx

```diff
@@ -0,0 +1,67 @@
+import { ArrowDownIcon, ArrowUpIcon } from '@chakra-ui/icons'
+import { Flex, Table, Tbody, Td, Th, Thead, Tr, useColorModeValue } from '@chakra-ui/react'
+import { useMemo } from 'react'
+import { Column, useSortBy, useTable } from 'react-table'
+
+type ReactTableProps = {
+  columns: Column<any>[]
+  data: any[]
```

Comment:
can we slightly tighten this to an `object[]` as per what `useTable()` takes as an option?